elasticsearch/native/src/main/java/org/apache/metamodel/elasticsearch/nativeclient/ElasticSearchDataContextFactory.java

```diff
@@ -131,10 +135,16 @@ public DataContext create(DataContextProperties properties, ResourceFactoryRegis
     }
 
     private Client createTransportClient(DataContextProperties properties) {
-        final Settings settings = Settings.builder().put().put("name", "MetaModel").put("cluster.name", getCluster(properties)).build();
+        final Settings settings = Settings
+                .builder()
+                .put("name", "MetaModel")
```

Comment:
Is the name anything but a label? If it is, it probably needs to be configurable. Maybe it should be anyway, since MetaModel is just a library for the actual client. If anyone is interested in a label for their client (e.g. for logging purposes, connection lists, branding, whatever), I guess we should let them.

Reply:
(OTOH, it's not something new, so it's probably out of the scope of this. Nevermind!)

Reply:
As you noticed, it was already there and it's part of the "native client", which is going to be deprecated as part of this Pull Request, so I'd rather not change it.